utils/prettier/prettier-config.json

```diff
@@ -6,13 +6,13 @@
     {
       "files": "*.ts",
       "options": {
-        "tabWidth": 4
+        "tabWidth": 1
```

Comment:
Please revert this file, The format I chose is fine for me, and it must remain unchanged.

Reply:
Oopsie, was not meant to commit this one, tried to get prettier to work, but wasn't able to. My apologies.